src/app/models/reservation.js

```diff
@@ -0,0 +1,11 @@
+import Backbone from 'backbone';
+
+const Reservation = Backbone.Model.extend({
+  url: function() {
+    console.log('https://ada-backtrek-api.herokuapp.com/trips/'+this.trip_id+'/reservations');
```

Comment:
It might make more sense to use `urlRoot` like so:

```
urlRoot() {
  return `https://ada-backtrek-api.herokuapp.com/trips/${this.get('trip_id')}/reservations`;
}
```
It's also preferred style to [use interpolation](https://github.com/airbnb/javascript#es6-template-literals).